samples/storage_csm_powerflex_v230.yaml

```diff
@@ -36,11 +36,13 @@ spec:
     sideCars:
     # comment the following section if you don't want to run the monitoring sidecar
```

Comment:
could you remove this comment and add comment similar to storage_csm_powerflex_v240.yaml